manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/plugins/NetworkPoliciesHandler.java

```diff
@@ -68,9 +68,14 @@
             // All Network Traffic Rules Go Under Allow Traffic
             List<NetworkTrafficRule> networkTrafficRules = serviceSpec.get(HyscaleSpecFields.allowTraffic, new TypeReference<>() {
             });
-            logger.info("Started Network Policies Handler");
+
+            // Only Port Number required to generate traffic rules
+            networkTrafficRules.stream().forEach(rule ->
+                    rule.getPorts().stream().forEach(port ->
+                            rule.getPorts().set(rule.getPorts().indexOf(port), port.split("/")[0])));
```

Comment:
Can the second iteration be done through map instead of forEach?
Something like:
networkTrafficRules.stream().forEach(rule -> rule
                    .setPorts(rule.getPorts().stream().map(port -> port.split("/")[0]).collect(Collectors.toList())));